src/main/webapp/resources/js/apis/users/users.js

```diff
@@ -1,7 +1,47 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 import axios from "axios";
 import { setBaseUrl } from "../../utilities/url-utilities";
+import moment from 'moment';
```

Comment:
Remove this import as it is not being used

Reply:
Please see 65cda97198979cf319f8bd3add44216589f94012.